content/en/docs/Examples/configuration_files.md

```diff
@@ -0,0 +1,181 @@
+| ``grid_ref`` | A dictionary containing the type of grid, and a list of grid variables defining the mapping from the domain file to NEMO file. |
+| ``chunks`` | A dictionary defining a dask [chunk shape](https://docs.dask.org/en/latest/array-chunks.html), used when loading in data files. JSON doesn't support integer keys, and so the dimensions name should be provided as the key instead. An empty dictionary will result in auto chunking. |
+| ``dataset``   | Parent key for holding configuration specific to the dataset files. |
+| ``domain`` | Parent key for holding configuration specific to domain files. This is an __optional__ key depending on whether a domain file is required or not.     |
+| ``dimension_map`` | Child key of dataset/domain. A dictionary defining the mappings between input data dimension names and the framework's standardised dimension names.  |
+| ``variable_map`` | Child key of dataset/domain. A dictionary defining the mappings between input data variable names and the framework's standardised variable names. |
+|``static_variables``| Parent key for holding configuration used for merging domain variables into the main dataset. |
+|``not_grid_vars``| Child key of static_variables. A list of grid independant variables to pull across from the domain file. |
+|``coord_vars``| Child key of static_variables. A list of variables to be set as coordinates after the merge. (Coordinates are reset and then set again as part of the merge.) |
+|``delete_vars``| Child key of static_variables. A list of variables to drop following the merge of domain and dataset. |
+| ``processing_flags`` | A list of strings referring to any preliminary processing methods to be carried out on the data. |
+
+### Indexed configuration
+| Key     | Description |
+| --- | ----------- |
+| ``type`` | A string value representing the type of configuration file. In the case of indexed config this will always be "indexed".       |
+| ``dimensionality`` | An integer value representing the number of dimensions within the data files.       |
+| ``chunks`` | A dictionary defining a dask [chunk shape](https://docs.dask.org/en/latest/array-chunks.html), used when loading in data files. JSON doesn't support integer keys, and so the dimensions name should be provided as the key instead. An empty dictionary will result in auto chunking. |
+| ``dataset``   | Parent key for holding configuration specific to the dataset files. |
+| ``dimension_map`` | Child key of dataset. A dictionary defining the mappings between input data dimension names and the framework's standardised dimension names.  |
+| ``variable_map`` | Child key of dataset. A dictionary defining the mappings between input data variable names and the framework's standardised variable names. |
+| ``processing_flags`` | A list of strings referring to any preliminary processing methods to be carried out on the data. |
+
+### Example configuration file
+Below is the template of a gridded configuration file:
```

Comment:
where "ex[number]" represents a static variable name that is hardwired into the package. E.g. "temperature".


(I find the use of non-standard variable names to demonstrate standard variable names a bit misleading. This caveat might clarify the situation)